amy/settings.py

```diff
@@ -131,6 +131,7 @@
     'rest_framework',
     'api',
     'captcha',
+    # 'debug_toolbar',
```

Comment:
I understand, but please don't check this into the code base. (Can we add a command-line config variable so that when you launch AMY locally for testing, debug_toolbar is enabled?)